pages/android/bugreport.md

```diff
@@ -0,0 +1,9 @@
+# bugreport
+
+> Show an Android bug report.
+> This command can only be used through `adb shell`.
+> More information: N/A.
```

Comment:
```suggestion
> More information:<https://android.googlesource.com/platform/frameworks/native/+/master/cmds/bugreport/>.
```

Reply:
I don't think that this one contains any useful information.

Reply:
Also agreed

Reply:
Hm, even though it will probably dissapoint everyone hoping to find more information, it also shows that there aren't more information, so I think we should include it.

Reply:
I'd say a licence would be good enough for more information